locale/zh_CN/manageiq.po

```diff
@@ -9907,9 +9910,15 @@ msgstr "ConfigurationScriptBase | 输出"
 msgid "ConfigurationScriptBase|Payload"
 msgstr "ConfigurationScriptBase | 有效内容"
 
+msgid "ConfigurationScriptBase|Payload error"
+msgstr "ConfigurationScriptBase | 有效内容错误"
+
 msgid "ConfigurationScriptBase|Payload type"
 msgstr "ConfigurationScriptBase | 有效内容类型"
 
+msgid "ConfigurationScriptBase|Payload valid"
+msgstr "ConfigurationScriptBase | 有效内容"
```

Comment:
ugh, looks like we had spaces after base in the existing translations... probably why they did the same here.

Reply:
fixed